tests/suites/test_suite_ssl.function

```diff
@@ -157,9 +156,14 @@ typedef struct mbedtls_test_message_queue
     int capacity;
 } mbedtls_test_message_queue;
 
-
 /*
  * Setup and free functions for the message metadata queue.
+ *
+ * \p capacity describes the number of message metadata chunks that can be held
+ *    within the queue.
+ *
+ * \retval  0, if a metadata queue of a given length can be allocated.
+ * \retval  MBEDTLS_ERR_SSL_ALLOC_FAILED, if allocation failed.
```

Comment:
This comment should be added in "Add a message metadata queue in ssl tests", not "Add a message-based socket mock connection to the ssl tests"